mapper/mapper.go

```diff
@@ -35,6 +35,8 @@ var dimensionTitleTranslator = map[string]string{
 func CreateFilterOverview(dimensions []filter.ModelDimension, filter filter.Model, dst dataset.Model, filterID, datasetID, releaseDate string) filterOverview.Page {
 	var p filterOverview.Page
 
+	log.Debug("mapping api response models into filter overview page model", nil)
```

Comment:
it feels like logging out some of the data might be useful here - but without wanting to log out the whole page. Possibly the filter ID and dataset ID?